documentation/modules/ROOT/pages/transformations/vitess-use-local-vgtid.adoc

```diff
@@ -0,0 +1,80 @@
+:page-aliases: configuration/vitess-use-local-vgtid.adoc
+[id="vitess-use-local-vgtid"]
+= Vitess Use Local VGTID
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+The vitess-specific `UseLocalVgtid` SMT is used to transform the VGTID from a list of several shards VGTIDs to a list of a single shard's VGTID for the shard that received the change for this event.
+
+It is done by using the `shard` field of the `source` of the event to determine the shard that received the change. Then that shard value is used to reduce the VGTID to only the relevant shard.
+
```

Comment:
```suggestion
The following example shows how to add the SMT to the connector configuration so that it converts the list of VGTIDs to the value of the local VGTID.
```